physics/GFS_rrtmgp_cloud_mp.F90

```diff
@@ -1,10 +1,5 @@
-!> \file GFS_rrtmgp_cloud_mp.F90
-!!
-!> \defgroup GFS_rrtmgp_cloud_mp GFS_rrtmgp_cloud_mp.F90
-!!
-!! \brief This module contains the interface for ALL cloud microphysics assumptions and 
-!! the RRTMGP radiation scheme. Specific details below in subroutines.
-!!
+! ########################################################################################
```

Comment:
I'm curious why this Doxygen markup was removed since the contents of this file don't really change much.

Reply:
Fixed.